src/custom/utils/operator/error.ts

```diff
@@ -40,7 +39,7 @@ export default class OperatorError extends Error {
 
   // Status 400 errors
   // https://github.com/gnosis/gp-v2-services/blob/9014ae55412a356e46343e051aefeb683cc69c41/orderbook/openapi.yml#L563
-  static apiErrorDetails = API_ERROR_CODE_DESCRIPTIONS
+  static apiErrorDetails = ApiErrorCodeDetails
```

Comment:
After this change, do you need `static apiErrorDetails` at all? Can't you just use the enum?

Reply:
can just use enum, yep

Reply:
noted in #751